airbyte-webapp/src/packages/cloud/views/auth/components/PersonQuoteCover/PersonQuoteCover.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import styles from "./PersonQuoteCover.module.scss";
+
+export interface PersonQuoteCoverProps {
+  overlayGradientStyle?: React.CSSProperties;
+  backgroundImageStyle?: React.CSSProperties;
+  quoteText?: string;
+  quoteTextStyle?: React.CSSProperties;
+  logoImageSrc?: string;
+  quoteAuthorFullName?: string;
+  quoteAuthorFullNameStyle?: React.CSSProperties;
+  quoteAuthorJobTitle?: string;
+  quoteAuthorJobTitleStyle?: React.CSSProperties;
+}
+export const PersonQuoteCover: React.FC<PersonQuoteCoverProps> = ({
+  backgroundImageStyle,
+  overlayGradientStyle,
+  quoteText,
+  quoteTextStyle,
+  logoImageSrc,
+  quoteAuthorFullName,
+  quoteAuthorFullNameStyle,
+  quoteAuthorJobTitle,
+  quoteAuthorJobTitleStyle,
+}) => {
+  return (
+    <div className={styles.container}>
+      <div className={styles.image} style={backgroundImageStyle} data-testid="background-image" />
+      <div className={styles.overlay} style={overlayGradientStyle} data-testid="gradient-overlay" />
+      <div className={styles.contentContainer}>
+        <blockquote className={styles.quote} style={quoteTextStyle} data-testid="quote">
+          <p>{quoteText ? quoteText : <FormattedMessage id="login.quoteText" />}</p>
+        </blockquote>
+        <img
+          src={logoImageSrc ? logoImageSrc : "/images/testimonials/cartdotcom-logo.svg"}
+          className={styles.companyLogo}
+          alt="company logo"
```

Comment:
Since this is just for decoration and not really important except for design, we can just change this to `alt=""` to hide it from screen readers.